Jesse - Refactor

Zilarrezko opened the pull request:
Merging from main to refactor changes from main, will merge back into main when changes are complete

Geoff will be mainly orchestrating the changes recently in main that he added.

Zilarrezko committed:
Merge branch 'main' into jesse

Zilarrezko committed:
Checkpoint Commit (Non functioning)

geoffcorvera committed:
fix ticket ids in /api/tickets endpoint

Zilarrezko committed:
Fix bug

geoffcorvera committed:
typo fixes

Zilarrezko committed:
Fix bug

Tickets weren't being added to cart. Thanks Geoff!

Zilarrezko committed:
Small Naming Refactor

Zilarrezko committed:
Fix Doorlist

Also naming refactor
Also fixed some spacing in the headers... I don't remember that being a problem before.

geoffcorvera committed:
Address create events merge conflict

geoffcorvera committed:
Merge branch 'jesse' of https://github.com/wpapsco/capstone-mockup into jesse

geoffcorvera committed:
Fix archive events

Zilarrezko committed:
Remove Todo

geoffcorvera committed:
Cleanup edit event pg & handle non-ok response from server

geoffcorvera reviewed (approved):
Quickly tested & confirmed:

1. Tests are passing
2. Can create event
3. Can edit event details (but not "showings")
4. Can archive event
5. Can load events & add tickets to cart

Added tiny amounts of error handling.

Zilarrezko committed:
Admin Panel Change

Since newsletter creation wasn't a priority, it was removed.
In it's place I added a link to download email subscription lists
